client/src/js/components/bhStockExpired/bhStockExpired.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <bh-grid-loading-indicator
+  loading-state="$ctrl.loading">
+</bh-grid-loading-indicator>
+
+
+<table
+ng-show = "$ctrl.expiredInventories.length > 0"
+class="table table-condensed table-bordered">
+<thead>
+  <tr>
+    <th colspan="2">
+      <span class="text-danger" 
+        translate-values="{ total : $ctrl.expiredInventories.length }"
+        translate="{{ 'STOCK.STOCK_EXPIRED_WARNING' }}">
+      </span>
+    </th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <th translate>FORM.LABELS.LOT</th>
+    <th translate>FORM.LABELS.INVENTORY</th>
+  </tr>
+  <tr ng-repeat = "inventory in $ctrl.expiredInventories | limitTo : 5">
+    <td>{{inventory.code}}</td><td>{{inventory.text}} ({{inventory.expiration_date}})</td>
+  </tr>
+
+  <tr>
+    <td colspan="2">
+      <span class="text-danger" 
+        ng-show="($ctrl.expiredInventories.length - 5) > 0"
+        translate-values="{ left : ($ctrl.expiredInventories.length - 5)}"
+        translate="{{ 'STOCK.AND_MORE' }}">
+      </span>
+
+    <a ui-sref= "stockLots({ filters : [
+      { key : 'period', value : 'allTime'},
+      { key : 'includeEmptyLot', value : 1  },
```

Comment:
Let's make this `includeEmptyLot` to be `0`.  We don't want to see lots that have expired if they do not have stock in them.